dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -48,54 +63,133 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     fn from(document: mongodb::ordered::OrderedDocument) -> Self {
         let account_id = document.get_i32("_id").unwrap() as u16;
         let batch_index = U256::from(document.get_i32("batchIndex").unwrap());
+        let valid_from_auction_id = U256::from(document.get_i32("validFromAuctionId").unwrap());
         StandingOrder {
             account_id,
             batch_index,
+            valid_from_auction_id,
             orders: document
                 .get_array("orders")
                 .unwrap()
                 .iter()
                 .map(|raw_order| raw_order.as_document().unwrap())
                 .map(|order_doc| super::Order {
-                        account_id,
-                        buy_token: order_doc.get_i32("buyToken").unwrap() as u8,
-                        sell_token: order_doc.get_i32("sellToken").unwrap() as u8,
-                        buy_amount: order_doc.get_str("buyAmount").unwrap().parse().unwrap(),
-                        sell_amount: order_doc.get_str("sellAmount").unwrap().parse().unwrap(),
-                    }
-                ).collect()
+                    account_id,
+                    buy_token: order_doc.get_i32("buyToken").unwrap() as u8,
+                    sell_token: order_doc.get_i32("sellToken").unwrap() as u8,
+                    buy_amount: order_doc.get_str("buyAmount").unwrap().parse().unwrap(),
+                    sell_amount: order_doc.get_str("sellAmount").unwrap().parse().unwrap(),
+                })
+                .collect(),
+        }
+    }
+}
+
+impl From<Arc<Log>> for StandingOrder {
+    fn from(log: Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        // Get basic data from event
+        let account_id = u16::pop_from_log_data(&mut bytes);
+        let batch_index = U256::pop_from_log_data(&mut bytes);
+        let valid_from_auction_id = U256::pop_from_log_data(&mut bytes);
+        let mut packed_orders_bytes = bytes.clone();
```

Comment:
No need for now since it's not implemented. 

I was going to use the `pop_from_log_data` methods that mutate the vars. So this lines shadow the `bytes` and make it mutable, so we can extract the params of the orders.

The thing I found doing this is that we don't have a `uint96` version of those pop methods...
Maybe is best just to take the slices from the Vec and getting the values in an immutable way without using those utils